ignitionLogic.py

```diff
@@ -0,0 +1,84 @@
+import pandas as pd
+import sys
+file_name = sys.argv[1]

```

Comment:
try to get more values from argument
like itration count, sensor added or not, file name,  device type.
and rename filename to arguments.
and try to name varaibles in camelCasing.